src/OeUI/OemrUI.php

```diff
@@ -455,21 +533,20 @@ private function headerActionJs($arrAction = array())
             // getItem() returns a string which is why we have to check for the string 'true'
             const elementTitle = localStorage.getItem('display#$page');
             let shouldDisplay = false;
-            if (typeof hideTitle != 'undefined' &&
-                (elementTitle == hideTitle || elementTitle == null)) {
+            if (typeof hideTitle != 'undefined' && (elementTitle == hideTitle || elementTitle == null)) {
                 shouldDisplay = true
             }
 
             // We display if we remember we're showing it, but we don't intentionally hide it (no else here to hide)
             // Because the hideaway is probably shown by default for a reason like in the billing manager
             if (shouldDisplay) {
-                $('.hideaway').show(500);
-                $('#show_hide').removeClass(showActionClass);
-                $('#show_hide').addClass(hideActionClass);
+                document.querySelector('.hideaway').classList.remove('d-none');
+                elementIcon.classList.remove(showActionClass);
+                elementIcon.classList.add(hideActionClass);
```

Comment:
elementIcon is never defined I can see. It is defined within the show_hide click() handler so it would be out of scope elsewhere.
I don't see how this would work in any case because while  show_hide click will set as soon as routine is exited then you lose scope. elementIcon needs to be assigned/declared somewhere global.